config.yaml

```diff
@@ -2,18 +2,22 @@
 # See LICENSE file for licensing details.
 
 options:
-  git-sync::repo:
+  git_sync_repo:
     type: string
     description: URL to repo to sync against
-  git-sync::branch:
+  git_sync_branch:
     type: string
     description: branch to sync
```

Comment:
Some more docu love in there configs :-) The content is right, but it feels like a little more polish in capitalization and how the sentences are constructed will make it feel nicer.

Reply:
Done.